controllers/appmesh/cloudmap_controller.go

```diff
@@ -70,7 +70,7 @@ func (r *cloudMapReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Named("cloudMap").
 		For(&appmesh.VirtualNode{}).
 		Watches(&k8s.NotificationChannel{Source: r.podEventNotificationChan}, r.enqueueRequestsForPodEvents).
-		WithOptions(controller.Options{MaxConcurrentReconciles: 3}).
+		WithOptions(controller.Options{MaxConcurrentReconciles: 1}).
```

Comment:
I changed this because I was receiving reconcile errors for duplicate update requests for the cloudmap services.

Reply:
It would be good to understand the root cause of these errors. Increasing concurrency to > 1 means concurrent reconciliation runs but not concurrent reconciliation of the same CRD. So, is there something in our model that is causing two CRD updates to converge to one cloudmap resource ?